js/src/ui/Tooltips/Tooltip/tooltip.js

```diff
@@ -30,15 +31,15 @@ let tooltipId = 0;
 
 class Tooltip extends Component {
   static propTypes = {
-    title: PropTypes.string,
-    text: PropTypes.string,
-    right: PropTypes.bool,
+    className: PropTypes.string,
     currentId: PropTypes.number,
     maxId: PropTypes.number,
-    className: PropTypes.string,
+    right: PropTypes.bool,
```

Comment:
Funnily enough, I played with that some weeks ago - it does some nice stuff, e.g. required first, then normal, then handlers (`on`Something).